src/lib/messenger.js

```diff
@@ -23,6 +23,19 @@ export default function messenger(bot) {
       const text = message.text.toLowerCase();
       const user = message.user || {};
 
+      function stripText(input) {
```

Comment:
I would prefer something like `stripCommand` since that's what the function is going to do, strip a command from the input given to it, and that's all necessary to know about it. 

I'm not sure it's a good idea to associate function names with constants elsewhere in the code, think of functions more like things that should be able to be run in isolation, unaware of other context and they still should make somewhat sense.